### PR TITLE
[risk=no] public database version change

### PR DESCRIPTION
**Pull request by SrushtiGangireddy**
Changed the public db version to use the latest version.

### CODE REVIEW

File: api/config/cdr_versions_prod.json [hunk 1]
```diff
@@ -10,6 +10,6 @@
     "releaseNumber": 1,
     "numParticipants": 946237,
     "cdrDbName": "cdr20181023",
-    "publicDbName": "public20181015"
+    "publicDbName": "public20181026"
```
Comment (calbach) [inline api/config/cdr_versions_prod.json]: This doesn't exist in prod yet. Are these changes breaking, or can prod stay on the old one for now?

File: api/config/cdr_versions_test.json [hunk 2]
```diff
@@ -22,6 +22,6 @@
     "releaseNumber": 2,
     "numParticipants": 946237,
     "cdrDbName": "cdr20181023",
-    "publicDbName": "public20181022"
+    "publicDbName": "public20181026"
```
Comment (calbach) [inline api/config/cdr_versions_test.json]: What about staging?

Comment (SrushtiGangireddy) [reply api/config/cdr_versions_test.json]: Updated and pushed the change.

File: api/config/cdr_versions_prod.json (hunk 1 above)
Comment (SrushtiGangireddy) [reply api/config/cdr_versions_prod.json]: I guess i would leave it as it is for now since databrowser is not functional in prod anyway.

**Pull request merged**